server/notebooks/names.py

```diff
@@ -0,0 +1,26 @@
+import random
+
+ELEMENTS = [
+    ("oxygen", "oxide"),
+    ("hydrogen", "hydride"),
+    ("argon", "argide"),
+    ("phosphorous", "phosphide"),
+]
+PREFIXES = ["di", "tri", "tetra", "penta", "hexa"]
+
+
+def _get_elements(num, elements):
+    i = int(random.random() * len(elements))
+    element_part = elements[i][num]
+    prefix_part = PREFIXES[int(random.random() * len(PREFIXES))]
```

Comment:
I don't know if you like it better but there's also `random.choice(PREFIXES)`

Reply:
I didn't know about that, thanks!